core/error.go

```diff
@@ -112,4 +114,19 @@ var (
 
 	// ErrBlobTxCreate is returned if a blob transaction has no explicit to field.
 	ErrBlobTxCreate = errors.New("blob transaction of type create")
+
+	// -- EIP-7702 errors --
+
+	// Message validation errors:
+	ErrEmptyAuthList         = errors.New("EIP-7702 transaction with empty auth list")
+	ErrSetCodeTxCreate       = errors.New("EIP-7702 transaction cannot be used to create contract")
+	ErrAuthSignatureVeryHigh = errors.New("EIP-7702 authorization with R or S value greater than 2^256 - 1")
+
+	// EIP-7702 state transition errors:
+	// Note these are just informational, and do not cause tx execution abort.
```

Comment:
These should not be defined here. This scope of erors are

```
// List of evm-call-message pre-checking errors. All state transition messages will
// be pre-checked before execution. If any invalidation detected, the corresponding
// error should be returned which is defined here.
```

Reply:
makes sense, thanks